include/dsn/dist/replication/replication.types.h

```diff
@@ -469,6 +469,14 @@ namespace dsn { namespace replication {
     {
         int32_t app_id;
         int32_t pidx;
+        inline bool operator < (const global_partition_id& r) const
```

Comment:
This will be overwritten by next time's code generation. That's why we move the comparison functions to the non-code-generated file.